src/qappfw.js

```diff
@@ -334,11 +350,23 @@ class QRadar {
         {
             throw new Error("You must supply an AQL string");
         }
-
-        return QRadar.windowOrTab(
-            "do/ariel/arielSearch?appName=Surveillance&pageId=FlowList&dispatch=performSearch" +
-            "&value(timeRangeType)=aqlTime&value(searchMode)=AQL" +
-            "&value(aql)=" + encodeURIComponent(aql), openWindow === false ? "FLOWVIEWER" : null);
+        // Creates variables for fetching qradar version, removing the dots and taking the first 6 characters from it, i.e. 202164
+        var get_qradar_version = top.QRADAR_VERSION.replaceAll(".","");
```

Comment:
This code duplicates the logic from above, would be better if there was a single function that checked the qradar version, for example `isUP4AndAbove()` that you could just use directly in your if statements?